packages/transaction-manager/test/unit/transactions-factory.test.ts

```diff
@@ -70,6 +70,57 @@ describe('transaction-factory', () => {
       ).toBe(true);
     }, 10000);
 
+    it('can create encrypted transaction with EthereumPrivateKeyCipherProvider', async () => {
+      const encryptedTx = await TransactionsFactory.createEncryptedTransactionInNewChannel(
+        data,
+        [
+          TestData.idRaw1.encryptionParams,
+          TestData.idRaw2.encryptionParams,
+          TestData.idRaw3.encryptionParams,
+        ],
+        TestData.fakeEpkCipherProvider,
+      );
+      // eslint-disable-next-line no-magic-numbers
+
+      if (encryptedTx.encryptedData) {
+        // eslint-disable-next-line no-magic-numbers
+        // 'encryptedData not right'
+        expect(encryptedTx.encryptedData.length).toBe(126);
+        // 'encryptedData not right'
+        expect(encryptedTx.encryptedData.slice(0, 2)).toEqual(
+          MultiFormatTypes.prefix.AES256_GCM_ENCRYPTED,
+        );
+      } else {
+        fail('encryptedData should not be undefined');
+      }
```

Comment:
You can use expect(encryptedTx.encryptedData).toBeDefined() or something like that instead of an if/else.